package.json

```diff
@@ -31,8 +31,10 @@
         "uuid": "^8.3.2"
     },
     "devDependencies": {
+        "@nomiclabs/hardhat-ethers": "^2.2.3",
```

Comment:
why not use the toolbox and remove this and gas reporter?

Reply:
I removed this, I tried to install toolbox and remove gas reporter but ran into a lot of dependency conflicts between the toolbox ethers and ethers in the dependencies. It seems like we are only using the gas reporter package from the toolbox so maybe importing the entire toolbox at the point is unnecessary?